aiohttp/helpers.py

```diff
@@ -703,6 +708,7 @@ class HeadersMixin:
 
     _content_type = None
     _content_dict = None
+    _content_disposition = False
```

Comment:
Quite strange default. Why `False`?

Reply:
False if header are not parsed/cached yet
None if there was an attempt to parse, but no header found
ContentDisposition if header was found and parsed